src/components/Main.js

```diff
@@ -13,10 +13,6 @@ import ToolBar from './ToolBar';
 
 import '../css/main.css';
 
-// Megahack to include font-awesome.
-injectCSS('https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css');
```

Comment:
@fernandojsg I think I can add `font-awesome` and `Roboto` (as already done) using web font.